README.md

```diff
@@ -405,3 +405,8 @@ Consider submitting to the [deno.land/x](https://deno.land/x/) registry.
 ### Darija
 
 - [A first look at Deno | BlaBlaConf 2021 🇲🇦](https://www.youtube.com/watch?v=Y_etUvzAa4s) 
+
+### Kurdish (Central)
+
+- [deno.land](https://deno-ckb.vercel.app)
```

Comment:
The link url and link text might better to be `https://deno.roj.im/`?

Reply:
If it changes in the future (for example if the translations transferred to another person), the vercel link will still work.

Reply:
Ok. let's keep it as is